content/pages/constitution.md

```diff
@@ -112,7 +114,9 @@ computing.
     4. To track the completion of all duties assigned to members of the
        committee;
 
-    5. To maintain the non-financial records of the committee.
+    5. To manage all non-computing-related volunteers;
```

Comment:
"Non-computing-related" sounds a little awkward to me. Also a little vague? 
Is there a reason you did not write "Non-wics-sys volunteers" ?